code/game/machinery/_machinery.dm

```diff
@@ -577,14 +577,15 @@ Class Procs:
 /obj/machinery/proc/can_be_overridden()
 	. = 1
 
-/obj/machinery/zap_act(power, zap_flags, shocked_objects)
-	. = ..()
+/obj/machinery/zap_act(power, zap_flags)
 	if(prob(85) && (zap_flags & ZAP_MACHINE_EXPLOSIVE) && !(resistance_flags & INDESTRUCTIBLE))
 		explosion(src, 1, 2, 4, flame_range = 2, adminlog = FALSE, smoke = FALSE)
 	else if(zap_flags & ZAP_OBJ_DAMAGE)
 		take_damage(power/2000, BURN, "energy")
 		if(prob(40))
 			emp_act(EMP_LIGHT)
+		power -= power / 2000
+	. = ..(power, zap_flags)
```

Comment:
```suggestion
	return ..()
```
Arguments are passed by default, and yes, the modifications carry on.

Reply:
That's really good to know, thank you